test/Microsoft.Extensions.ProjectModel.Tests/MsBuild/MsBuildProjectDependencyProviderTests.cs

```diff
@@ -165,6 +164,9 @@ public void BuildDependenciesForProject()
                 Assert.NotNull(mvcPackage);
 
                 Assert.True(mvcPackage.Dependencies.Any(dependency => dependency.Name.Equals("Microsoft.Extensions.DependencyInjection", StringComparison.OrdinalIgnoreCase)));
+
+                Assert.True(context.ProjectReferences.Any());
+                Assert.True(context.ProjectReferences.First().EndsWith("Library1.csproj"));
```

Comment:
Can you assert that if finds the full path? It should equal `Path.Combine(fileProvider.Root, "Library1/LIbrary1.csproj")`